_support-articles/windows/special-characters

```diff
@@ -0,0 +1,19 @@
+---
+layout: article
+title: "Fixing crashes related to 'special' characters in Windows usernames"
+last_updated: 2019-01-28 13:00:00 +0000
+---
+
+Having an exclamation point (!), or some non-English Alphanumeric characters in your Windows Username (local windows user account) will cause the game to crash.
+
+In some cases you can fix this by changing the working directory the game uses by following the instructions [here](/support-articles/changing-working-direcotry/).
```

Comment:
https://bugs.mojang.com/browse/MCL-9929 shows that this is not fixed by setting --workdir

Reply:
Its not just for that issue. It covers all bases